contributor_docs/unit_testing.md

```diff
@@ -139,3 +139,31 @@ If you need to add a new test file, add it to that folder, then add the filename
 When you add a new test, running `npm test` will generate new screenshots for any visual tests that do not yet have them. Those screenshots will then be used as a reference the next time tests run to make sure the sketch looks the same. If a test intentionally needs to look different, you can delete the folder matching the test name in the `test/unit/visual/screenshots` folder, and then re-run `npm test` to generate a new one.
 
 To manually inspect all visual tests, run `grunt yui:dev` to launch a local server, then go to http://127.0.0.1:9001/test/visual.html to see a list of all test cases.
+
+
+The visual test environment is set up to execute your commands sequentially rather than running a preload or draw function that you provide.
+In continuous integration (CI) environments, it's crucial to keep tests running as quickly as possible. Running a full `preload/draw` cycle as in a regular p5.js sketch can significantly slow down the testing process.
```

Comment:
It's not quite that the preload/draw cycle takes long, but we want to encourage the code to be as minimal as possible, which generally means not drawing multiple frames when just one will do, using a small canvas size, and only loading assets if required by the functionality being tested. We maybe don't have to capture all of that in this one sentence, but maybe we can rephrase slightly?

Reply:
In a continuous integration (CI) environment, optimizing test speed is essential. It is advantageous to keep the code concise, avoid unnecessary frames, minimize canvas size, and load assets only when essential for the specific functionality under test. 
@davepagurek Let me know if the above paragraph would make more sense or something else.